### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by S-BLH**
- created a repo by the name of cash register
- cloned it to my pc
- created html,css and js files
- added the linters
- added html , css and java script codes
- added a design to the code
- cleaned the code

### CODE REVIEW

**Comment by latifa-wakili** [comment]
✔ Created a professional Repo
✔ Correct use of Gitflow 
✔ Commit messages are meaningful
✔ Setup the linters correctly 
✔ No linter errors (All linters checks have been passed 🟢)
✔ Created the pull request 
✔ Good PR description
✔Your code is clean
✔Your repo has a professional README.md file
✔Your design is responsive
keep it up!

**Comment by S-BLH** [comment]
@ahzamir ostad kindly review my project, as my partner commented and reviewed it already.

**Comment by ahzamir** [comment]
> ✔ Created a professional Repo ✔ Correct use of Gitflow ✔ Commit messages are meaningful ✔ Setup the linters correctly ✔ No linter errors (All linters checks have been passed 🟢) ✔ Created the pull request ✔ Good PR description ✔Your code is clean ✔Your repo has a professional README.md file ✔Your design is responsive keep it up!

@latifa-wakili 

- Why do you give feedback without checking your partner's code and project there are lots of issues in this project but you approved it.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+          background-color: navy;
+          color: white;
+          border: none;
+          cursor: pointer;
+        }
+
+        button:hover {
+          background-color: purple;
+        }
+    </style>
```
Comment (ahzamir) [inline index.html]: - You have a different file for your CSS codes which is `style.css` and here also you linked a file named `styles.css` please change the name of your `style.css` to `styles.css` and move your CSS code there.

**Review by ahzamir — CHANGES_REQUESTED** [review]
@S-BLH 

- Your project has some issus that I will point out please also check the comments below for more required changes:
  - Your project should show the amount of money in different cashes like this [example](https://cash-register.freecodecamp.rocks/):
  - Your design:  
image
  - Freecodecamp design:
image

**Comment by S-BLH** [comment]
@ahzamir Ostad as you said, I changed the whole appearance of the project, and added the numbers to it, kindly review it again.

**Comment by ahzamir** [comment]
@S-BLH 

- Your project is not responsive and also I can see the total price for the item as shown in the example
- also when I enter a cash value and click on purchase it is not showing me anything

**Comment by S-BLH** [comment]
@ahzamir OStad did the things you said, kindly review it.

**Comment by ahzamir** [comment]
@S-BLH 

- Don't mention me again you have to get your partner feedback before mentioning me

**Comment by S-BLH** [comment]
@ahzamir ostad my partner already commented on this project, she didnt comment on the pokemon project, and also she is not responding right now to my texts,

**Comment by ahzamir** [comment]
@S-BLH 

- As i said before you have to show everything for the user but I can not see the total piece:

image

**Comment by ahzamir** [comment]
- And also when user add less money than the price you should show the error.

**Comment by S-BLH** [comment]
@ahzamir Dear Ostad, this time everything is clear and I solved its problem, kindly review it

File: index.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+            .touch-row {
+                flex-direction: column;
+            }
+
+            .key {
+                width: 100%;
+            }
+        }
+    </style>
```
Comment (ahzamir) [inline index.html:126]: - I told you in previous comments to separate your styling code from your HTML file as you have a CSS file for them.
- Solve this issue and then directly merge your branch.

**Review by ahzamir — APPROVED** [review]
### Hi @S-BLH 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Cash Register project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**